src/plugins/mavlink_ftp/crc32.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <sys/types.h>
```

Comment:
What's `sys/types.h` for?

Reply:
It was for size_t. Replaced it with std::size_t.